### ISSUE TITLE
OpenSearch error while running `post-upgrade.sh` in a 4.1.2 to 4.1.3 upgrade

### ISSUE THREAD

**Issue by yacdaniel** [craftercms/craftercms#6503]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [x] The issue is in the develop branch
- [X] The issue is in the latest released 4.1.x
- [ ] The issue is in the latest released 4.0.x
- [ ] The issue is in the latest released 3.1.x

### Describe the issue

Authoring/Delivery upgrade from 4.1.2 to 4.1.3 completes without errors in the `upgrade-target.sh` script. Then the user is asked to run `post-upgrade.sh`, during this proccess an error is thrown in the terminal that initiated the script. This error cause that the user can't see clearly the step of 
```
Please make sure Crafter has started successfully before continuing
> Continue? [(Y)es/(N)o]
```

### Steps to reproduce

Steps:

1. Create a site in Authoring & Delivery 4.1.2
2. Run the `upgrade-target.sh` in both environment. 
3. See the issue

### Relevant log output

```shell
Please make sure Crafter has started successfully before continuing
> Continue? [(Y)es/(N)o]: uncaught exception in thread [main]
java.lang.IllegalArgumentException: index template [ss4o_metrics_template] has index patterns [ss4o_metrics-*-*] matching patterns from existing templates [ss4o_metric_template] with patterns (ss4o_metric_template => [ss4o_metrics-*-*]) that have the same priority [1], multiple index templates may not match during index creation, please use a different priority
        at org.opensearch.cluster.metadata.MetadataIndexTemplateService.addIndexTemplateV2(MetadataIndexTemplateService.java:558)
        at org.opensearch.cluster.metadata.MetadataIndexTemplateService$4.execute(MetadataIndexTemplateService.java:491)
        at org.opensearch.cluster.ClusterStateUpdateTask.execute(ClusterStateUpdateTask.java:65)
        at org.opensearch.cluster.service.MasterService.executeTasks(MasterService.java:874)
        at org.opensearch.cluster.service.MasterService.calculateTaskOutputs(MasterService.java:424)
        at org.opensearch.cluster.service.MasterService.runTasks(MasterService.java:295)
        at org.opensearch.cluster.service.MasterService$Batcher.run(MasterService.java:206)
        at org.opensearch.cluster.service.TaskBatcher.runIfNotProcessed(TaskBatcher.java:204)
        at org.opensearch.cluster.service.TaskBatcher$BatchedTask.run(TaskBatcher.java:242)
        at org.opensearch.common.util.concurrent.ThreadContext$ContextPreservingRunnable.run(ThreadContext.java:849)
        at org.opensearch.common.util.concurrent.PrioritizedOpenSearchThreadPoolExecutor$TieBreakingPrioritizedRunnable.runAndClean(PrioritizedOpenSearchThreadPoolExecutor.java:282)
        at org.opensearch.common.util.concurrent.PrioritizedOpenSearchThreadPoolExecutor$TieBreakingPrioritizedRunnable.run(PrioritizedOpenSearchThreadPoolExecutor.java:245)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:840)
For complete error details, refer to the log at /opt/crafter/bundles/412auth/crafter/logs/search/opensearch.log

```
https://gist.github.com/yacdaniel/4051caca7b0231b78090e03166945468

**Comment by yacdaniel** [comment]
Verified in `4.1.3-SNAPSHOT`

**Comment by Euquimides** [comment]
Hi! Updated and rechecked this issue. Fix verified for v4.2.0. Thank you! Closing this ticket.
